docker-compose.yml

```diff
@@ -29,3 +29,10 @@ services:
     ports:
       - 4317:4317 # OTLP gRPC receiver
       - 4318:4318 # OTLP http receiver
+  localstack:
```

Comment:
Man I don't love having to run a localstack Docker instance all the time, but I can't think of a better solution.

Reply:
Is this the first AWS Service Provider we're implementing? Are integration tests the only option or is mocking/ faking palatable?

Reply:
We try to use "real"(ish) systems for our integration tests, but fakes in unit tests.